### PR TITLE
manual QA notes for interims

### PR DESCRIPTION
**Pull request by KrupaPammi**

### CODE REVIEW

File: qa/manualQA.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+**What is Manual Testing?**
+
+Manual testing means testing of an application manually by a human. A tester or QA performing manual testing for ensuring that a website or an application is working properly by various conditions which are written in test cases. 
+Manual testing is the process of finding the Bug, Defect or issues in the Website. The aim of a software tester or QA is how to break the system. The tester also checks the UI design, functionality, and performance of the website by clicking on various elements.
+
+
+
+**The Objective of Testing** 
+
+To ensure that the application meets the business and user requirements
```
Comment (pvhee) [inline qa/manualQA.md]: maybe use bullet points too, as done below?

File: qa/manualQA.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+There are four different breakpoints
+
+LG - Large Desktops
+
+MD – Desktops and Tablets landscape
+
+SM – Tablets Portrait view and Mobiles Landscape
+
+XS – Mobile Portrait View
```
Comment (pvhee) [inline qa/manualQA.md]: needs indenting

**Review by pvhee — CHANGES_REQUESTED** [review]
Looks good, apart from some cosmetic changes needed

File: qa/manualQA.md (hunk 1 above)
Comment (KrupaPammi) [reply qa/manualQA.md]: added

File: qa/manualQA.md (hunk 2 above)
Comment (KrupaPammi) [reply qa/manualQA.md]: fixed

**Review by pvhee — APPROVED** [review]

**Pull request merged**